lua/fpp/server/ownability.lua

```diff
@@ -267,51 +267,58 @@ end
 Networking
 ---------------------------------------------------------------------------]]
 util.AddNetworkString("FPP_TouchabilityData")
--- Sends 32 + 32 + 5 + 20 = 89 bits of ownership data per entity
+-- Sends 13 + 8 + 5 + 20 = 46 bits of ownership data per entity
 local function netWriteEntData(ply, ent)
-    -- EntIndex for when it's out of the PVS of the player
-    net.WriteUInt(ent:EntIndex(), 32)
+    -- EntIndex for when it's out of the PVS of the player`
+    net.WriteUInt(ent:EntIndex(), 13)
 
     local owner = ent:CPPIGetOwner()
-    net.WriteUInt(IsValid(owner) and owner:EntIndex() or -1, 32)
+    net.WriteUInt(IsValid(owner) and owner:EntIndex() or -1, 8)
     net.WriteUInt(ent.FPPRestrictConstraint and ent.FPPRestrictConstraint[ply] or ent.FPPCanTouch[ply], 5) -- touchability information
     net.WriteUInt(ent.FPPConstraintReasons and ent.FPPConstraintReasons[ply] or ent.FPPCanTouchWhy[ply], 20) -- reasons
 end
 
-function FPP.plySendTouchData(ply, ents)
-    local count = #ents
-
-    if count == 0 then return end
+local touchabilityDataQueue = {}
+local function startNetWriteQueue()
+    if timer.Exists( "FPP_TouchabilityQueue" ) then return end
+
+    timer.Create( "FPP_TouchabilityQueue", 0, 0, function()
```

Comment:
This would add a delay of one tick for any data that gets added to the queue. Previously it would send the data immediately. I'm not sure if that will have a noticeable effect when e.g. the server is slow.

Reply:
I personally couldn't notice it, the reason the one tick is there is to catch multiple calls that would change the data,  https://github.com/FPtje/Falcos-Prop-protection/pull/325#issuecomment-1975209906 currently FPP sends two net messages for 1 prop spawn because it sets multiple values on it, this 1 tick delay aims to solve that and other addons that possibly change ownership data.